modules/machineset-yaml-gcp.adoc

```diff
@@ -80,10 +80,10 @@ endif::infra[]
             labels: null
             sizeGb: 128
             type: pd-ssd
+          gcpMetadata:
+            creationTimestamp: null
```

Comment:
the format of this block is actually different, and shouldn't contain the `creationTimestamp`. this field has the format:

```
gcpMetadata:
  - key: SomeMetadataKey
    value: "some value for the key"
```

and these will get set in GCP for the instances you are creating.